third_party/libtock-drivers/src/nfc.rs

```diff
@@ -23,25 +24,26 @@ mod allow_nr {
     pub const RECEIVE: usize = 2;
 }
 
+pub enum TransOrRecvStatus {
+    Error,
```

Comment:
`Error` is too unspecific. Maybe make it clear that it is supposed to include all errors that are neither `InvalidBuffer` nur `OOM`?